### PR TITLE
[chromecast] Complete list of available things and channels in README

### PR DESCRIPTION
**Pull request by engineergreen**
Add the full list of available Things and Channels to the documentation for the Chromecast binding. 

This change is to documentation only.

### CODE REVIEW

File: addons/binding/org.openhab.binding.chromecast/README.md [hunk 1]
```diff
@@ -12,7 +12,12 @@ This can be configured on the binding level:
 
 ## Supported Things
 
-The binding currently supports the "classic" Chromecast that is an HDMI dongle as well as the Chromecast Audio, which only does audio streaming and offers a headphone jack.
+| Things             | Description                     | Thing Type |
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.chromecast/README.md]: Please reformat the tables.

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍

File: addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastCommander.java [hunk 2]
```diff
@@ -87,9 +87,6 @@ public void handleRefresh() {
             status = chromeCast.getStatus();
             statusUpdater.processStatusUpdate(status);
 
-            if (status == null) {
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastCommander.java]: Why did you change this?

File: addons/binding/org.openhab.binding.chromecast/README.md [hunk 3]
```diff
@@ -12,7 +12,12 @@ This can be configured on the binding level:
 
 ## Supported Things
 
-The binding currently supports the "classic" Chromecast that is an HDMI dongle as well as the Chromecast Audio, which only does audio streaming and offers a headphone jack.
+| Things             | Description                                                                  | Thing Type |
+|--------------------|------------------------------------------------------------------------------|------------|
+| Classic Chromecast | The HDMI dongle type Chromecast                                              | chromecast |
```
Comment (kubawolanin) [inline addons/binding/org.openhab.binding.chromecast/README.md]: `Google Home`, `Google Home Mini` and `Google Home Max` are also supported as they are auto-discovered as `chromecast`

**Comment by engineergreen** [comment]
Sorry about that, I was debugging another problem and accidentally committed to the wrong branch.  I've fixed that now.  Thanks for your time.

File: addons/binding/org.openhab.binding.chromecast/README.md (hunk 3 above)
Comment (martinvw) [reply addons/binding/org.openhab.binding.chromecast/README.md]: @engineergreen can you take a look at @kubawolanin 's comment as well.

**Comment by martinvw** [comment]
Sorry for only noting it now but can you please sign off your commit, see the following:

Squash your commits and change commit message to include a `Signed-off-by` (to certify that you wrote it or otherwise have the right to pass it on as an open-source), see http://docs.openhab.org/developers/contributing/contributing.html#sign-your-work

For squashing / rebasing see:
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

File: addons/binding/org.openhab.binding.chromecast/README.md (hunk 3 above)
Comment (kubawolanin) [reply addons/binding/org.openhab.binding.chromecast/README.md]: I beliece @engineergreen already did, check  https://github.com/openhab/openhab2-addons/pull/3058/commits/71bb00ff08a3989b97ab2ba2563b505a1d2251dd#diff-8d75b964f954f65f11950d506642292fR17

Comment (martinvw) [reply addons/binding/org.openhab.binding.chromecast/README.md]: I was looking for a new row :-) Thanks!

**Comment by engineergreen** [comment]
@martinvw Not a problem, I'm new to this.  Done.
